app/bt/Home/Home.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
```

Comment:
I am a bit worried about separating for the Home screens for GPS and BT. What are the ramifications around design inconsistencies and such?

Reply:
totally agree, but its a bit hard to consolidate them quickly as the underlying data model is currently very strategy dependent and it would take a lot of effort to massage it out of the main screen.  I think it would great to move towards consolidating the screens over time though.